themes/login-forms/templates/Security.ss

```diff
@@ -18,6 +18,8 @@
                     <% if not $SiteConfig.Title %>SilverStripe<% end_if %>
                 </h1>
             </header>
+            <% if $Message %><div class="$MessageType">$Message</div><% end_if %>
+            <% if $Content && $Content != $Message %><div>$Content</div><% end_if %>
```

Comment:
Do you want to add a class name to this div for user's project styling if they want to do it?

Reply:
Added BEM selectors